.ci/test_r_package_windows.ps1

```diff
@@ -165,7 +165,7 @@ if ($env:COMPILER -ne "MSVC") {
 } else {
   $env:TMPDIR = $env:USERPROFILE  # to avoid warnings about incremental builds inside a temp directory
   $INSTALL_LOG_FILE_NAME = "$env:BUILD_SOURCESDIRECTORY\00install_out.txt"
-  Run-R-Code-Redirect-Stderr "source('build_r.R')" *> $INSTALL_LOG_FILE_NAME ; $install_succeeded = $?
+  Run-R-Code-Redirect-Stderr "source('build_r.R')" 1> $INSTALL_LOG_FILE_NAME ; $install_succeeded = $?
```

Comment:
Oh, wait, does it mean that we lose all not success logs that are written not to stdout?

Reply:
We already lost any of them by the changes from #3168 , I think. This step will still break if something is broken, but to get better logs  we'd have to re-run with this set to `Rscript build_r.R` instead of using `Run-R-Code-Redirect-Stderr`.

I know that's bad, but the main thing is that we know the build will fail if a chance breaks the R package. I hope that https://github.com/PowerShell/PowerShell/issues/12823 will get resolved and GitHub Actions will be updated soon, then maybe we can remove the workaround to suppress stderr